### PR TITLE
ci(PT13113): Add github workflow for code style

### PR DESCRIPTION
**Pull request by DennisGarding**

### CODE REVIEW

File: .php-cs-fixer.cache [hunk 1]
```diff
@@ -0,0 +1 @@
...de.","separate":"bottom","comment_type":"PHPDoc"},"no_useless_else":true,"no_useless_return":true,"general_phpdoc_annotation_remove":{"annotations":["copyright","category"]},"phpdoc_var_annotation_correct_order":true,"doctrine_annotation_indentation":true,"doctrine_annotation_spaces":true,"PhpCsFixerCustomFixers\/no_useless_comment":true,"PhpCsFixerCustomFixers\/single_space_after_statement":true,"PhpCsFixerCustomFixers\/single_space_before_statement":true,"PhpCsFixerCustomFixers\/phpdoc_param_type":true},"hashes":{"Bootstrap.php":"9773adf256f5c633a01467b5927936be","php-cs-fixer.php":"5ad3e0cf67fb470c9204e782953ddcd1","Subscriber\/BackendIndex.php":"8f8d66312309679ac2e58d5b319a239b","Subscriber\/Frontend.php":"04983fe496911fb7fa5575dcc3344176","Components\/Paypal\/RestClient.php":"8d7954786bb9e980b00f33078658378a","Components\/Paypal\/Client.php":"033c6f8efa666979f84492a543618902","Components\/Paypal\/AddressValidator.php":"165bb645f002ee2137e5c76aa0627089","Components\/CSRFWhitelistAware.php":"6ca982b75150407436cfee786d015ecb","Controllers\/Frontend\/PaymentPaypal.php":"cb73a3500a2479d66aff037974602ba1","Controllers\/Backend\/PaymentPaypal.php":"99917342c7a5ad97e40cbaa318854b8e"}}
```
Comment (mitelg) [inline .php-cs-fixer.cache]: please add that to gitignore

Comment (DennisGarding) [reply .php-cs-fixer.cache]: This is a draft.. it has long been done

File: .github/workflows/php-code-analysis.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    pull_request:
+    workflow_dispatch:
+    push:
+        branches:
+            - main
+
+jobs:
+    call-analyse-workflow:
+        name: Analyse code for SwagPaymentPaypal
+        uses: rathmerdominik/docker-images-testing/.github/workflows/php-code-analysis.yml@sw-27107/fix-workflows-for-shopware-and-plugins
```
Comment (mitelg) [inline .github/workflows/php-code-analysis.yml]: why not the official workflow?

File: .github/workflows/javascript-code-analysis.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+on:
+    pull_request:
+    workflow_dispatch:
+    push:
+        branches:
+            - main
+
+jobs:
+    javascript-code-analysis:
```
Comment (mitelg) [inline .github/workflows/javascript-code-analysis.yml]: this should be in the reusable workflow, see https://shopware.atlassian.net/browse/PT-13120

**Review by mitelg — CHANGES_REQUESTED** [review]

File: .github/workflows/php-code-analysis.yml (hunk 2 above)
Comment (rathmerdominik) [reply .github/workflows/php-code-analysis.yml]: @swDennis  has to change this
This was for debug purposes and should never be used in production

It is imperative though that https://github.com/shopware5/docker-images-testing/pull/30 gets merged first **before** this gets merged

File: Makefile
Comment (rathmerdominik) [inline Makefile:1]: Nice one

this goes to @shopware5/shopware-core 
If you see any plugin without a Makefile include one at the next pull request with the basic commands
These provide a nice and easy to use standard

**Review by mitelg — APPROVED** [review]

**Review by ennasus4sun — APPROVED** [review]

**Pull request merged**